Update Documentation. Add documentation about ellipses (#4172)
openvinotoolkit/cvat: site/content/en/docs/manual/advanced/annotation-with-ellipses.md

---
title: 'Annotation with ellipses'
linkTitle: 'Annotation with ellipses'
weight: 11
description: 'Guide to annotating tasks using ellipses.'
---

It is used for road sign annotation etc.

> Before starting, you need to select the `ellipse` on the controls sidebar.

Reviewer: ```suggestion
First of all you need to select the `ellipse` on the controls sidebar.
```

![](/images/image239.jpg)

> Choose a `Label` and click `Shape` or `Track` to enter drawing mode. An ellipse is created in the same way as

Reviewer: ```suggestion
Choose a `Label` and click `Shape` or `Track` to start drawing. An ellipse can be created the same way as
```

> a [rectangle](/docs/manual/advanced/annotation-with-rectangles/), you need to specify two opposite rectangle points,

Reviewer: ```suggestion
a [rectangle](/docs/manual/advanced/annotation-with-rectangles/), you need to specify two opposite points,
```

> and the ellipse will be inscribed in a rectangle. Press `N` or click the `Done` button on the top panel

Reviewer: ```suggestion
and the ellipse will be inscribed in an imaginary rectangle. Press `N` or click the `Done` button on the top panel
```

to complete the shape.

![](/images/image240_mapillary_vistas.jpg)

> You can rotate the created ellipse using a rotation point in the same way as

Reviewer: ```suggestion
You can rotate ellipses using a rotation point in the same way as
```
